frontend/src/templates/Field/ChildrenCompound/ChildrenCompoundField.tsx

```diff
@@ -200,9 +207,13 @@ const ChildrenBody = ({
   myInfoChildrenBirthRecords,
   isSubmitting,
   formContext,
+  error,
 }: ChildrenBodyProps): JSX.Element => {
   const { register, getValues, setValue, watch } = formContext
 
+  // Child name error will always be the first element of the error array
+  // const childNameError = error ? error[0] : undefined
```

Comment:
We can also remove this comment

Reply:
[a6afd16](https://github.com/opengovsg/FormSG/pull/6735/commits/a6afd16cc2398ff529772e99ce829bfc6a080356)